Longest_Abecedarian_Word/Program.cs

```diff
@@ -4,9 +4,54 @@ namespace Longest_Abecedarian_Word
 {
     class Program
     {
+
```

Comment:
Unnecessary Blank lines, remove them. Use them only to select control structures, eg if ... else, switch case, and all type of cycles.